roles/scripts/files/wp-list-env

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+##
+# Gives user easy way to environment variables which are important for him
+##
+
+# Source environment if doesn't exist yet
+if [ -z $PS1 ]; then
+  source /etc/container_environment.sh
+fi
+
+IFS='%' read -a envvars <<< "$LIST_CONTAINER_ENVS"
```

Comment:
The file is there and the contents looks sane. It is probably used somewhere. What do wp-list-env script does will not affect anything else than make users able to run this command as promised in docs.wp-palvelu.fi

```
vagrant@wp-dev:~$ ll /etc/conta*
-rw-rw-r-- 1 vagrant vagrant 387 Feb 22 10:20 /etc/container_environment.sh
vagrant@wp-dev:~$ cat /etc/container_environment.sh 
# Just some helpers to make this look like the production environment
export DB_NAME=vagrant
export DB_USER=vagrant
export DB_PASSWORD=vagrant
export WP_PROJECT=/data/wordpress
export HTDOCS=$WP_PROJECT/htdocs
export WEB_ROOT=$HTDOCS
export WP_CORE=$HTDOCS/wordpress
export WP_CONTENT=$HTDOCS/wp-content
export WP_ENVIRONMENT=development

export HTTPS_DOMAIN_ALIAS=pointscene.seravo.dev

```